http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonPayloadIT.java

```diff
@@ -74,71 +71,68 @@ public void sendLargeComplexJSONPayload() throws IOException {
                 .statusCode(CREATION_STATUS_CODE)
                 .body(containsString("619 Rockaway Parkway, Hillsboro, California, 7507"));
 
-        getApp().given()
+        Response response = getApp().given()
                 .get("/persons")
                 .then()
                 .statusCode(OK_STATUS_CODE)
-                .body(containsString("Hello, Singleton Middleton! You have 4 unread messages."));
+                .body(containsString("Hello, Singleton Middleton! You have 4 unread messages.")).extract().response();
 
     }
 
     @Test
-    public void sendJSONWithEmptyFieldsPayload() throws IOException {
+    public void sendJSONWithEmptyFieldsPayloadNoResourceValidation() throws IOException {
         String jsonPayload = readJSONFile("football-empty-teams.json");
-
-        getApp().given()
+        Response response = getApp().given()
                 .contentType(ContentType.JSON)
                 .body(jsonPayload)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
-                .body(containsString("Some fields are empty or null"));
+                .statusCode(CREATION_STATUS_CODE).extract().response();
+        JsonPath jsonPath = response.jsonPath();
+        // Check that there are 2 teams with empty colorTShirt fields
+        assertThat(jsonPath.getList("teams.findAll { it.colorTShirt == '' }"), hasSize(2));
     }
 
     @Test
     public void sendInvalidJSONPayload() throws IOException {
         String jsonPayload = readJSONFile("football-teams_invalid.json");
-
         getApp().given()
                 .contentType(ContentType.JSON)
                 .body(jsonPayload)
                 .when()
                 .post("/football/upload-football-json")
                 .then()
-                .statusCode(BAD_REQUEST_STATUS_CODE)
+                .statusCode(INTERNAL_SERVER_ERROR_STATUS_CODE)
                 .body(containsString(
-                        "Invalid JSON - JsonProcessingException : Unexpected character ('{' (code 123)): was expecting comma to separate Array entries"));
+                        "Internal error: Invalid token=CURLYOPEN at (line no=7, column no=3, offset=81). Expected tokens are: [COMMA]"));
```

Comment:
Will be this error always the same? I'm thinking like check only for part of this error as I think `(line no=7, column no=3, offset=81)` can change base of the input.

Reply:
Yes, it will be the same, when the payload/json file is kept the same.

Reply:
Afaik looking at the error from CI 
```Expected: a string containing "Internal error: Invalid token=CURLYOPEN at (line no=7, column no=3, offset=81). Expected tokens are: [COMMA]"
  Actual: {"stack":"Internal error: Invalid token=CURLYOPEN at (line no=7, column no=3, offset=87). Expected tokens are: [COMMA]"}